src/content/docs/errors-inbox/apm-tab.mdx

```diff
@@ -0,0 +1,103 @@
+---
+title: 'APM: Group errors tab '
+tags:
+  - APM
+  - APM UI pages
+  - Error analytics
+metaDescription: New Relic's error profiles feature in the classic APM view shows trends and anomalies that help you compare and troubleshoot error events.
```

Comment:
what does 'classic APM view' mean here? we might try to avoid that.

Reply:
fixed!